web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.test.tsx

```diff
@@ -117,7 +117,7 @@ test('startAgent re-throws errors that are thrown while spawning the process', a
   expect(error).toBeInstanceOf(AgentProcessError);
   expect(result.current.currentAction).toStrictEqual({
     kind: 'start',
-    attempt: makeErrorAttempt(AgentProcessError.name),
+    attempt: makeErrorAttempt(new AgentProcessError()),
```

Comment:
In production code (`currentAction.attempt.statusText !== AgentProcessError.name`) we compare an error using `.name` property. Do we want to refactor it here?